README.md

```diff
@@ -64,7 +64,8 @@ You can collect them like this:
 Ferm::Rule <<| tag == 'allow_kafka_server2server' |>>
 ```
 
-You can also define rules in Hiera. Make sure to use `alias()` as interpolation function, because `hiera()` will always return a string.
```

Comment:
To my understanding every sentence in markdown file should be exactly one line (without character limit) so changes are more visible in a diff. An alternative would be to split sentences at logical delimiters like `,`.

Reply:
our linter wants a maxium line length of 80 characters, hence the newline.

Reply:
Ok, I can live with that